conda_smithy/configure_feedstock.py

```diff
@@ -365,8 +365,11 @@ def finalize_config(config, platform, forge_config):
     """For configs without essential parameters like docker_image
     add fallback value.
     """
-    if platform.startswith("linux") and not "docker_image" in config:
-        config["docker_image"] = [forge_config["docker"]["fallback_image"]]
+    if platform.startswith("linux"):
+        if "docker_image" in config:
+            config["docker_image"] = config["docker_image"][0]
```

Comment:
Can you make this a list? There's no reason to change this. (I think conda-build will complain as well)

Reply:
I'm confused. Won't we need to index the value in the CI scripts then?

Reply:
What I would like to see is that https://github.com/conda-forge/zlib-feedstock/pull/35 will be a no-op. Can you rerender that PR with this branch again?

Reply:
Ok, thanks for clarifying. Will do. Have been doing this locally as we have been iterating.

Reply:
The latest commit makes re-rendering `zlib` a no-op.